### PR TITLE
adding `numpy_to_vtk_image_data` method to utility

### PR DESCRIPTION
**Pull request by ganimtron-10**
Adding a `numpy_to_vtk_image_data` method in utility to ease the image loading process.
Issue mentioned at #500 .
PTAL.
Thanks!

### CODE REVIEW

**Review by skoudoro — COMMENTED** [review]
Hi @ganimtron-10,

Overall, it looks good. Can you rebase your PR and address my comment? 

It should be ready to be merged after that.

Thanks !

**Comment by skoudoro** [comment]
Thank you for rebasing @ganimtron-10. Can you also address my comment above? (add spacing, origin, and deep as optional parameter)

**Comment by ganimtron-10** [comment]
Hi @skoudoro ,
I was just facing issues with my local workspace so it took time to reflect the changes!
Now I have updated the function with optional parameters.
Let me know if it has any changes!
Thanks !

File: fury/utils.py [hunk 1]
```diff
@@ -160,6 +160,43 @@ def numpy_to_vtk_cells(data, is_coords=True):
...
+    array : ndarray
+        pixel coordinate and colors values.
+
+    Returns
+    -------
+    vtk_image : vtkImageData
+
+    """
+    if array.ndim not in [2, 3]:
+        raise IOError("only 2D (L, RGB, RGBA) or 3D image available")
```
Comment (skoudoro) [inline fury/utils.py:178]: Can you add a test for this condition?  use `np.assert_raises`

File: fury/utils.py [hunk 2]
```diff
@@ -160,6 +160,43 @@ def numpy_to_vtk_cells(data, is_coords=True):
     return cell_array
 
 
+def numpy_to_vtk_image_data(array, spacing=(1.0, 1.0, 1.0),
+                            origin=(0.0, 0.0, 0.0), depth=1):
```
Comment (skoudoro) [inline fury/utils.py:164]: I was talking about the `deep` option and not `depth`. This is a boolean value. See below.

**Review by skoudoro — COMMENTED** [review]
Thank you for the update. See Below for the last comment. Thanks @ganimtron-10

File: fury/utils.py (hunk 2 above)
Comment (ganimtron-10) [reply fury/utils.py]: Sorry for that, I just got confused between those!

**Comment by ganimtron-10** [comment]
Hey @skoudoro ,
I have addressed all the comments, PTAL!
Thanks.

**Comment by skoudoro** [comment]
Thanks @ganimtron-10! merging

**Pull request merged**